src/pages/resultsView/querySummary/QuerySummary.tsx

```diff
@@ -43,7 +43,7 @@ export default class QuerySummary extends React.Component<{ queryStore:QueryStor
 
     private get singleStudyUI() {
         return <div>
-            <h4><StudyLink study={this.props.store.studies.result[0]}/></h4>
+            <h4><StudyLink study={this.props.queryStore.selectedStudies[0]}/></h4>
```

Comment:
I'm not sure this makes sense.  selectedStudies in queryStore will change prior to user submitting the new query.  We don't want to this change just when user checks checkboxes.

Reply:
Yeah, True. Currenlt `ResultsViewPageStore` studies (`store.studies.result`) would just have predefined studies. we should even render virtual studies in this case.